robocopkaka/bootcamp-29-project
# #158441260 Remove unnecessary tests

#### What does this PR do?
remove unnecessary tests
#### Description of Task to be completed?
* remove unnecessary tests
* move `Signup.test.js` to the `common` sub-folder
* move `Login.test.js` to the `common` sub-folder
* move `Header.test.js` to the `common` sub-folder
* move `Footer.test.js` to the `common` sub-folder
* move `LoginButtons.test.js` out of the `common` sub-folder
* move `HomeButtons.test.js` out of the `common` sub-folder
#### How should this be manually tested?
* clone this repo
* `cd` into the project folder
* run `npm install` to install other dependencies
* run `npm install -g sequelize-cli` to install `sequelize` globally
* run `npm run test:jest` to run `frontend` tests
* coverage should have increased and no errors
#### Any background context you want to provide?
some of the previous tests didn't affect my test coverage in any way
#### What are the relevant pivotal tracker stories?(if applicable)
#158441260
